data/sql/updates/pending_db_world/rev_1660959750178228200.sql

```diff
@@ -0,0 +1,62 @@
+-- Add formation linking to stationary packs in AQ40
+DELETE FROM `creature_formations` WHERE `leaderguid` IN (87595, 87598, 87606, 87604, 87615, 87625, 87621, 87622, 88077, 87657, 87656, 87655, 87654);
```

Comment:
Use memberguid instead of leaderguid, memberguid is the primary key.

Reply:
Done 👍